javascript-modules-engine/tests/.env.example

```diff
@@ -6,4 +6,3 @@ JAHIA_URL=${JAHIA_URL:-http://jahia:8080}
 SUPER_USER_PASSWORD=${SUPER_USER_PASSWORD:-root1234}
 NEXUS_USERNAME=${NEXUS_USERNAME:-NEXUS_USERNAME}
 NEXUS_PASSWORD=${NEXUS_PASSWORD:-NEXUS_PASSWORD}
-CYPRESS_PREPACKAGED_SITE_URL="jar:mvn:org.jahia.samples/javascript-modules-samples-hydrogen-prepackaged/LATEST/zip!/site.zip"
```

Comment:
Are we sure we want to remove that variable? As an external URL it is subject to change, being able to set it from the job declaration may help at some point. 
If we want to remove this, we also need to remove its usage in the [compose file](https://github.com/Jahia/javascript-modules/blob/adjust-cypress-tests/javascript-modules-engine/tests/docker-compose.yml#L33)